corehq/apps/style/templates/style/crispy/multi_field.html

```diff
@@ -1,8 +1,14 @@
-<div class="form-group">
+<div class="form-group {{ multifield.css_class }}" {{ multifield.flat_attrs|safe }}>
     <label class="control-label {{ label_class }}">
         {{ label_html }}
     </label>
     <div class="controls {{ field_class }} controls-multiple">
-      {{ field_html }}
+        <div class="form-inline">
```

Comment:
`form-inline` isn't great here. I would actually call this `row` and give each field the `col-sm-*` (and related) classes